byob/modules/ransom.py

```diff
@@ -15,6 +16,7 @@
 # packages
 import Crypto.PublicKey.RSA
 import Crypto.Cipher.PKCS1_OAEP
```

Comment:
I just noticed, `Crypto.Cipher.PKCS1_OAEP` needs to updated to `Cryptodome.Cipher.PKCS1_OAEP` - if you could integrate this into your next commit I would appreciate it.